gradle-plugin/src/main/java/com/microsoft/gradle/GradleProjectModelBuilder.java

```diff
@@ -63,13 +63,20 @@ public boolean canBuild(String modelName) {
 	}
 
 	public Object buildAll(String modelName, Project project) {
+		int majorVersion = Integer.parseInt(project.getGradle().getGradleVersion().split("\\.")[0]);
```

Comment:
Is the value of `project.getGradle().getGradleVersion()` same as `GradleVersion.current()`?

If not, I think we can use

`GradleVersion.version(project.getGradle().getGradleVersion())` to do the comparison

Reply:
You are right, it makes more sense that way